CHANGELOG.md

```diff
@@ -6,17 +6,19 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
-## Added
+## Breaking Change
 
-- `RPMBuilder::source_date()` method for clamping modification time of files. This allows reproducible builds when included files are re-generated (having the same content but with different mtimes).
+- The `RPMBuilder::build_time` method is removed. Package build time is now included by default and can be clamped using the `RPMBuilder::source_date_epoch` method.
+- The `RPMPackage::sign` method now requires to pass timestamp used for signing.
 
-## Changed
+## Added
 
-- Build time is now included into package by default
+- `RPMBuilder::source_date_epoch` method for clamping modification time of files, build time of the package, and signature time. This functionality is required for reproducible generation of packages.
 
 ### Fixed
 
-- CentOS 7 support by using long sizes only for packages bigger than 4 GiB
+- CentOS 7 support by using long sizes only for packages bigger than 4 GiB.
+- Compatibility with `rpm --rebuilddb` by including package build time by default.
```

Comment:
Package build time still deserves an entry under `Added`

Reply:
Do you want both lines:
```md
### Added
- Package build time is now included by default.
### Fixed
- Compatibility with `rpm --rebuilddb` by including package build time by default.
```
?

Or only the former?

Reply:
Both, but the second line could be something more like 

> Avoid a longstanding bug in `rpm --rebuilddb` by adding a package build time by default.